compiler-cli/src/config.rs

```diff
@@ -12,6 +14,52 @@ pub fn root_config() -> Result<PackageConfig, Error> {
     read(paths.root_config())
 }
 
+/// Get the config for a dependency module. Return the config for the current
+/// project if a dependency doesn't have a config file.
+pub fn find_package_config_for_module(
+    mod_path: &str,
+    manifest: &Manifest,
+    project_paths: &ProjectPaths,
+) -> Result<PackageConfig, Error> {
+    let gleam_projects: Vec<&String> = manifest
+        .packages
+        .iter()
+        .filter(|package| package.build_tools.contains(&"gleam".to_string()))
+        .map(|package| &package.name)
+        .collect();
+
+    let package_path = fs::read_dir(project_paths.build_packages_directory()).map_or(None, |x| {
+        {
+            x.filter_map(Result::ok)
+                .filter(
+                    |project_folder| match project_folder.file_name().into_string() {
+                        Ok(utf8_str) => gleam_projects.contains(&&utf8_str),
+                        Err(_) => false,
+                    },
+                )
+                .find(|file| {
+                    let mut path = file.path();
+                    path.push("src");
```

Comment:
I *think* I got this working but I can't test because the `test` directory isn't included in gleam package builds. Since this code is only triggered on a dependency I don't think that feature needs to be supported cause of that.

Reply:
Oh you're completely right, there's no need to check for test inside packages. I'm a fool.

Would you like to remove the test checking bit? I can do that if you're prefer, don't wanna waste your time.

Reply:
I'll do it :)